### PR TITLE
configs/rtl8721csm/nettest: add MAKE_SAMSUNG_HEADER definition

### PR DESCRIPTION
**Pull request by an4967**
- add MAKE_SAMSUNG_HEADER definition

### CODE REVIEW

File: build/configs/rtl8721csm/nettest/defconfig [hunk 1]
```diff
@@ -229,7 +232,7 @@ CONFIG_AMEBAD_FLASH_PAGE_SIZE=256
 CONFIG_AMEBAD_FLASH_BLOCK_SIZE=4096
 CONFIG_AMEBAD_AUTOMOUNT=y
 CONFIG_AMEBAD_AUTOMOUNT_USERFS=y
-CONFIG_AMEBAD_AUTOMOUNT_USERFS_DEVNAME="/dev/smart0p7"
+CONFIG_AMEBAD_AUTOMOUNT_USERFS_DEVNAME="/dev/smart0p6"
```
Comment (sunghan-chang) [inline build/configs/rtl8721csm/nettest/defconfig:235]: Please revert this and below.

Comment (an4967) [reply build/configs/rtl8721csm/nettest/defconfig]: @sunghan-chang I reverted them as you said.

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Review by davidfather — DISMISSED** [review]
Looks good to me.

**Comment by sunghan-chang** [comment]
@an4967 Please resolve the conflict and please rewrite the title. There is no change on partition anymore.

**Comment by an4967** [comment]
> 
> 
> @an4967 Please resolve the conflict and please rewrite the title. There is no change on partition anymore.

I fixed the conflicts and the title

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**